src/client/apiclient.ts

```diff
@@ -118,37 +96,46 @@ const getApiData = async (config: AxiosRequestConfig): Promise<any> => {
  * @param companyNumber the company number.
  * @param token the bearer security token to use to call the api
  */
-export const getCompanyProfile = async (companyNumber: string, token: string): Promise<CompanyProfile> => {
-  const COMPANY_PROFILE_PATH = `${API_URL}/company/${companyNumber.toUpperCase()}/`;
+export const getCompanyProfile = async (companyNumber: string, token: string): Promise<ExtensionsCompanyProfile> => {
+  logger.debug("Creating CH SDK ApiClient");
+  const api = createApiClient(undefined, token, `${API_URL}`);
 
-  const config: AxiosRequestConfig = getBaseAxiosRequestConfig(token);
-  config.method = HTTP_GET;
-  config.url = COMPANY_PROFILE_PATH;
+  logger.info(`Looking for company profile with company number ${companyNumber}`);
+  const sdkResponse: Resource<CompanyProfile> =
+    await api.companyProfile.getCompanyProfile(companyNumber.toUpperCase()) as Resource<CompanyProfile>;
+
+  if (sdkResponse.httpStatusCode >= 400) {
+    throw {
+      status: sdkResponse.httpStatusCode,
+    };
+  }
+
+  logger.debug("Data from company profile SDK call " + JSON.stringify(sdkResponse, null, 2));
 
-  const data = await getApiData(config) as CompanyProfileResource;
+  const companyProfile: CompanyProfile = sdkResponse.resource as CompanyProfile;
```

Comment:
no need to add type to the `companyProfile` variable as you are casting it explicitly to `CompanyProfile`. you can simple omit it e.g. `const companyProfile = sdkResponse.resource as CompanyProfile`